weather_dl/download_pipeline/parsers.py

```diff
@@ -328,8 +328,12 @@ def require(condition: bool, message: str) -> None:
     num_partition_keys = len(partition_keys)
     if use_date_as_directory(config):
         num_partition_keys -= 1
-        if str(params.get('target_path')).endswith('/'):
-            params['target_path'] = params.get('target_path')[:-1]
+        try:
+            target_path = t.cast(str, params['target_path'])
+        except KeyError:
+            pass
```

Comment:
A more conventional method we use in our codebase is to call `get` with a default value. Would something like: 
```
target_path = t.cast(str, params.get('target_path', ''))
```
work here?

Reply:
Yep that should be fine. Updating the PR